aircraft/F450/F450-set.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+
```

Comment:
This file `F450-set.xml` should be removed as it is only meaningful to FlightGear.

Reply:
resolved with commit corrections